python-examples/README.md

```diff
@@ -4,8 +4,254 @@ Termhub in 5 Minutes: python Tutorial
+    * Run the command 'pip install -r requirements.txt' in a console window to check these libraries and install any
+      that are not already installed.
+    * If pip itself is not installed, run the command 'curl <https://bootstrap.pypa.io/get-pip.py> -o get-pip.py; python
+      get-pip.py' to install it.
+
+The various scripts make use of the `python-examples/config.ini` file to load necessary information that is uniform
+across all tests.
+
+When running the tests, make sure you are on the root folder.
+
+Sample Python Calls
+-----------------
+
+The following examples are exhibited by various unit tests defined in the code in `python-examples`.
+All commands to run these tests should be run from that directory.
+
+- [Login](#login)
+- [Get terminologies](#get-terminologies)
+- [Get project terminologies](#get-project-terminologies)
+- [Get specific terminology](#get-terminology)
+- [Export terminology](#export-terminology)
+- [Get concept by code](#get-concept-by-code)
+- [Get concept by code with explicit include parameter](#get-concept-by-code-with-include)
+- [Get concept relationships by code](#get-concept-relationships)
+- [Get concept inverse relationships by code](#get-concept-inverse-relationships)
+- [Get concept trees](#get-treepos)
+- [Find concepts by search term (use paging to get only first 5 results)](#find-concepts)
+- [Find concepts by search term with explicit include parameter](#find-concepts-include)
+- [Find concepts by search term and expression](#find-concepts-expr)
+- [Find terms by search term](#find-terms)
+- [Support autocomplete/typeahead for first few characters typed](#autocomplete)
+
+### Login [MUST BE RUN FIRST]
+
+Login and acquire an access token for a username and password.  
+You will need to set your environment variable for `USER_NAME` and `PASSWORD`.  
+The commands below will set a `token` variable equal to the access token. This token is used for the remaining tests.
+
+```
+pytest tests/test_login.py::TestLogin::test_login
+```
+
+Copy the `TOKEN` value and set this as an environment variable to run the remaining tests
+
+<a name="get-terminologies"/>
+
+### Get terminologies
+
+Return all loaded terminologies currently hosted by the API. This call also takes
+search parameters such as query, limit, offset, sort, and ascending to allow searching
+across available terminologies.
+
+```
+pytest tests/test_get_terminologies.py::TestGetTerminologies::test_get_terminologies
+```
+
+[Back to Top](#top)
+
+<a name="get-project-terminologies"/>
+
+### Get project terminologies
+
+Return all terminologies for the specified project identified by either projectId or projectLabel.
+
+```
+pytest tests/test_get_project_terminologies.py::TestGetProjectTerminologies::test_get_project_terminologies
+```
+
+[Back to Top](#top)
+
+<a name="get-terminology"/>
+
+### Get terminology
+
+Return a specific terminology by its terminologyId. The UUID below is an example
```

Comment:
below -> included

Reply:
Copied from the curl README. However, I moved this text to the test for better clarity.